fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/explain/ExplainPlanVisitor.java

```diff
@@ -512,13 +512,15 @@ public ExplainPlanVisitor visitSelectorPlan(@Nonnull final RecordQuerySelectorPl
     @Override
     public ExplainTokens visitStreamingAggregationPlan(@Nonnull final RecordQueryStreamingAggregationPlan streamingAggregationPlan) {
         visit(streamingAggregationPlan.getChild());
-        pipe().addKeyword("AGG").addWhitespace().addAliasDefinition(streamingAggregationPlan.getInner().getAlias())
+        pipe().addKeyword("AGG").addWhitespace().addPush()
+                .addCurrentAliasDefinition(streamingAggregationPlan.getInner().getAlias())
```

Comment:
When I read this, I was so confused for a moment, it took me a while to realize that both code bases are now in a single repo!!